doc/how-to/microcloud-with-terraform.md

```diff
@@ -0,0 +1,62 @@
+# Single Node Microcloud deployment (Nested VMs)
+
+## Prerequisites
+
+- Single Ubuntu node with LXD installed
+- A network of type bridged on the host, e.g. from lxc network list
+```
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+|      NAME      |   TYPE   | MANAGED |      IPV4       |           IPV6            | DESCRIPTION | USED BY |  STATE  |
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+| br0            | bridge   | NO      |                 |                           |             | 17      |         |
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+```
+
+## Clone the terraform
+
+Get the terraform from the [demos folder](../../demos/microcloud-deploy)
```

Comment:
Worth adding the actual `git clone https://github.com/canonical/microcloud` command and `cd` into `demos/terraform/*`?